runtime/tutor.txt

```diff
@@ -416,6 +415,296 @@ _________________________________________________________________
 
 
 
+
+=================================================================
+=                 3.5 SELECTING TO A CHARACTER                  =
+=================================================================
+
+ Type f<ch> to select up to and including (find) a character.
+ Type t<ch> to do the same, but not including ('til) a character.
```

Comment:
```suggestion
 Type t<ch> to do the same, but not including (till) a character.
```
Is 'til even valid?

Reply:
"'til" is abbreviated "until", which is correct.

Reply:
[Merriam-webster](https://www.merriam-webster.com/words-at-play/should-you-use-until-or-till-or-til) says:
> So there you have it: you will probably wish to avoid ’till, use ’til advisedly, and use both until and till freely. And if you use till in writing and someone tells you that you have made an error, simply take the extra L off the end of the word and poke them in the eye with it.

They are both used but it seems 'til is less looked down upon.

Reply:
Hmm actually https://writingexplained.org/til-or-till-difference seems very confident "till" is the correct word and "'til" isn't right at all.. I'll leave it as is for now as it seems a but controversial.